CoreScriptsRoot/Modules/StyleWidgets.lua

```diff
@@ -0,0 +1,40 @@
+--[[
+  Filename: StyleWidgets.lua
+  Written by: dbanks
+  Version 1.0
+  Description: Widgets with common style elements that can be shared across
+      different GUIS.
+--]]
+----------------- SERVICES ------------------------------
+local CoreGui = game:GetService("CoreGui")
+local RobloxGui = CoreGui:WaitForChild("RobloxGui")
+
+----------- UTILITIES --------------
+local utility = require(RobloxGui.Modules.Settings.Utility)
+
+----------- CLASS DECLARATION --------------
+--[[ Classes ]]--
+local StyleWidgets = {}
+
+----------- CONSTANTS --------------
+StyleWidgets.TabSelectionHeight = 6
+
+function StyleWidgets.MakeTabSelectionWidget(parent)
+	local tabSelection = utility:Create'ImageLabel'
+	{
+		Name = "TabSelection",
+		Image = "rbxasset://textures/ui/Settings/MenuBarAssets/MenuSelection.png",
+		ScaleType = Enum.ScaleType.Slice,
+		SliceCenter = Rect.new(3,1,4,5),
```

Comment:
I just did a cut and paste from 
CoreScriptsRoot/Modules/Settings/SettingsPageFactory.lua

See line 96 in the old file.

So maybe it should be but that could be an independent change.